samples/security/spire/README.md

```diff
@@ -0,0 +1,44 @@
+# Integrating SPIRE through Envoy SDS
+
+There are currently multiple forms of integrating third-party Certificate Authorities (CA) whitin Istio. This example deploys a setup of [SPIRE](https://github.com/spiffe/spire) (the SPIFFE Runtime Environment) as an Exernal CA, through [Envoy's SDS API integration](https://github.com/alexandrealvino/istio.io/blob/6eff1cd45dfce244978cafc10a88f7620c0438fc/content/en/docs/ops/integrations/spire/index.md).
+
+To assist this deployment, two additional components will be shipped together with SPIRE, the [CSI Driver](https://github.com/spiffe/spiffe-csi) and the [Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar).
```

Comment:
We could deploy this example without the CSI driver and use the easier daemonset/hostpath setup. This would mean less overhead for the quickstart, but we would be kind of  "teaching bad manners", I suppose? That's why I didn't want to use a simpler deployment setup. Let me know what you guys think about this @maxlambrecht @alexandrealvino @sanderson042

Reply:
I think the CSI Driver is a MUST, from the point of view of what I believe the istio maintainers prefer for the socket injection into the pods. The k8s workload registrar is a nice to have.